.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/prepare
       - name: Install Playwright Browsers
-        run: pnpm playwright install --with-deps
+        run: pnpm playwright install-deps chromium
```

Comment:
From https://github.com/seriouslysean/philly-js-club-website-remix/actions/runs/4833216939/jobs/8612951143, 

```plaintext
Run pnpm playwright install-deps chromium
Installing dependencies...
Switching to root user to install dependencies...
...
```

...which takes 22 seconds. It'd be nice to not have those 22 seconds of setting up the machines.

It's definitely much better than taking multiple minutes though! 😄